schemas/derivatives/SCHEMA_AMM (with vault)

```diff
@@ -0,0 +1,116 @@
+| pair_address            | The contract address of the pair.                         | string |
+| pair_symbol             | The symbol of the pair.                                   | string |
+| pair_liquidity          | The liquidity available for trading for the pair (24h rolling average, in tokens).  | string |
+| pair_liquidity_usd      | The liquidity available for trading for the pair (24h rolling average, in USD).  | string |
+| volume_usd              | The volume of each pair opened and closed in the given snapshot (24h rolling average, in USD). | number |
+| open_interest_longs_usd | The sum of open interest of longs held for each pair, in USD. | number |
+| open_interest_shorts_usd | The sum of open interest in shorts held for each pair, in USD. | number |
+| liquidation_fees_usd    | The total liquidation fees accrued for each pair (24h rolling average, in USD). | number |
+| funding_rate            | The funding rate for each pair at the time of the snapshot, as a percentage. | number |
+| open_fees_usd           | The total open fees accrued for each pair (24h rolling average, in USD). | number |
+| close_fees_usd          | The total close fees accrued for each pair (24h rolling average, in USD). | number |
+
+### LP Snapshot
+
+Liquidity providers snapshot (one entry for each token in a vault).
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| block_date               | The timestamp truncated (ie, YYYY-MM-DD format for daily snapshots and YYYY-MM-DD HH:00:00 for hourly snapshots). | date |
+| chain_id                 | Standard chain id.                                        | number |
+| vault_address            | The address of the vault.                                  | string |
+| lp_address               | The address of the liquidity provider.                    | string |
+| token_index              | The index of the token in the smart contract.             | number |
+| token_address            | The address of the token provided as liquidity.           | string |
+| token_symbol             | The symbol of the token.                                  | string |
+| amount_useful            | The amount of tokens supplied by the LP that are liquid and usable by users in the vault, decimal normalized. | number |
+| amount_useful_usd        | The amount of liquid tokens supplied in USD. | number |
+| liquidated_amount        | The amount of tokens liquidated from the LP (when the taker has positive PnL if a position is closed), decimal normalized. | number |
+| liquidated_amount_usd    | The amount liquidated, in USD.                            | number |
+| total_lp_fees_usd        | Total fees generated for the LP at the time of snapshot (in USD). | number |
```

Comment:
`total_lp_fees_usd` - is this the amount of fee that is earned by the individual LP provider?

Reply:
yes, it's in the LP snapshot, so for each LP

Reply:
this field seems difficult to calculate here, but we can leave it here and wait for team's feedback